### ISSUE TITLE
Development concept for the fork 

### ISSUE THREAD

**Issue by rfuehrer** [generaliinformatik/scrumonline#2]
Think about how we shape the development in this fork. Background: when we develop new features or changes, we want to put them upstream via PR. If the PR is not accepted, we should still be able to ensure that our adaptations can be adopted in our systems.

**Comment by dehesselle** [comment]
1. To avoid confusion and staying in-sync, every branch that is present in upstream is to be considered "off limits" for us, i.e. we mirror those branches and don't commit to them.
1. We always develop features/fixes/... in dedicated branches. We use these feature branches to make PRs to upstream.
1. If upstream merges our feature branch, we can pull that back in to our fork.
1. If upstream is not interested in our changes (or our changes are too specific to our needs), we create a new PR (with explanation why upstream isn't interested, including link to rejected PR) and merge that to our own "generali-master" (or "generali-develop" or whatever) branches.

Should we quickly diverge from upstream (because "4." above), maintaining a relationship and trying to stay somewhat in-sync becomes complicated or impossible. If that happens, we no longer make PRs to upstream.

**Comment by rfuehrer** [comment]
I agree with you. We should therefore define a default branch for us, which the colleagues who run scrumonline use as a reference. I think points 1,2 and 3 were already self-evident for us ;)
